scripts/MakefileImpl

```diff
@@ -63,6 +63,7 @@ clean:
 	find . -type d -name .current | xargs rm -rf
 	find . -type f -name 'current_build.h' | xargs rm -f
 	find . -type f -name 'current_build.h.*' | xargs rm -f
+	find . -type f -name '.current_std???.txt' | xargs rm -f
```

Comment:
`stdout`, `stderr`. Temporary outputs of a `make check` failed in the middle, which we forgot to remove in `make check`.